codestodians/food-basket-front-end
# [Feature: #158070068] Refactor LoginForm component

#### What does this PR do?
- Updates the `LoginForm` to use `react-apollo`'s `Mutation` component.

#### Description of Task to be completed?
- Update `react-apollo`.
- Refactor the `LoginForm` component to use the `Mutation` component from `react-apollo`.
- Deprecate the `Login-connect` module.
- Fix failing tests as a result of the absence of the `ApolloProvider`

#### How should this be manually tested?
- Check out into the `ft-use-mutation-component-in-login-158070068` branch.
- Install dependencies.
- Run the App and test that Login works as before.
- Run tests to see that all tests pass.

#### What are the relevant pivotal tracker stories?
- [#158070068](https://www.pivotaltracker.com/story/show/158070068)
